from_cpython/Lib/test/test_complex.py

```diff
@@ -476,8 +474,8 @@ def test_repr(self):
         self.assertEqual(repr(complex(0, -INF)), "-infj")
         self.assertEqual(repr(complex(0, NAN)), "nanj")
 
-    def test_neg(self):
-        self.assertEqual(-(1+6j), -1-6j)
+    # def test_neg(self):
+    #     self.assertEqual(-(1+6j), -1-6j)
 
```

Comment:
And same thing for the other changes to this file -- can we have `Pyston change` comments with them explaining why it differs from CPython.